kpops/component_handlers/kubernetes/model.py

```diff
@@ -12,7 +12,10 @@
 
 
 # JSON values
-Json: TypeAlias = dict[str, "Json"] | list["Json"] | str | int | float | bool | None
+JsonPrimitive: TypeAlias = (
+    dict[str, "Json"] | list["Json"] | str | int | float | bool | None
+)
+Json = TypeVar("Json", bound=JsonPrimitive)
```

Comment:
todo: move somewhere more generic

Reply:
I could see it living in `kpops.utils`

Reply:
yep, I improved this quite a bit and moved it to `kpops.utils.types`